core/domain/email_manager.py

```diff
@@ -184,7 +206,30 @@ def _send_email(
     Raises an Exception if the sender_id is not appropriate for the given
     intent. Currently we support only system-generated emails and emails
     initiated by moderator actions.
+
+    Args:
+        recipient_id: str. The user ID of the recipient.
+        sender_id: str. The user ID of the sender.
+        intent: str. The intent string for the email, i.e. the purpose/type.
+        email_subject: str. The subject of the email.
+        email_html_body: str. The body (message) of the email.
+        sender_email: str. The sender's email address.
+        bcc_admin: Boolean. Optional, default False.
```

Comment:
bcc_admin: bool. Whether to send a copy of the email to the admin's email address.

(Don't worry about specifying defaults here. They're obvious from the function signature.)